src/routes/submit/submit.js

```diff
@@ -40,7 +47,43 @@ export const submit = [
         reference
       )
 
-      await sendEmailToCaseWorker({ content: caseWorkerEmailContent })
+      let linkToFile = null
+      const fileAnswer = /** @type {FileAnswer} */ (
+        getQuestionFromSections(
+          'upload-plan',
+          'biosecurity-map',
+          request.payload?.sections
+        )?.answer
+      )
+
+      if (fileAnswer) {
+        if (String(fileAnswer.value?.skipped).toLowerCase() === 'true') {
```

Comment:
The value in the [API ought to be a boolean rather than a string](https://github.com/DEFRA/apha-apps-perms-analysis/blob/master/data-architecture/case-management-submission-schema.ts#L32), so we can just do: 

```javascript
if (fileAnswer.value?.skipped) { }
```